_docs/development/registration.md

```diff
@@ -42,9 +41,14 @@ module.exports = function(options = {}) {
 }
 ```
 
-Note that when registering a plugin that exposes a function instead of an object, you must execute the function when it is required so that it returns an object. Continuing on from the exported function example above:
+Lastly, some plugin types define constructor function with specific properties:
```

Comment:
Can you specify the `some plugin types` here?

Reply:
Let me know what you think about this: https://github.com/koopjs/koopjs.github.io/pull/70/commits/62b998910976c2634f38087b159775b2dcd0aa07.

Reply:
Looks good :+1: